File: python/src/test/TortillaMaizTest.py
import unittest
from TortillaMaiz import *

class TortillaMaizTest(unittest.TestCase):
    def setUp(self):
        self.tortillaMaiz = TortillaMaiz()

    def testCurrentTemperature(self):
        self.tortillaMaiz.setCurrentTemperature(21)
        self.assertEqual(21,self.tortillaMaiz.getCurrentTemperature())

    def testFalseToast(self):
        self.tortillaMaiz.toast(False)
        self.assertFalse(self.tortillaMaiz.isToasted())

    def testTrueToast(self):
        self.tortillaMaiz.toast(True)
        self.assertTrue(self.tortillaMaiz.isToasted())

    def testToasting(self):
        self.assertEqual(10,self.tortillaMaiz.getToastTemperature())

File: csharp/unittest-practice/src/test/TortillaMaizTest.cs
using NUnit.Framework;
using unittestpractice.main;

namespace unittestpractice.test
{
    [TestFixture]
    internal class TortillaMaizTest
    {
        private TortillaMaiz _tortillaMaiz;

        [SetUp]

        public void Setup()
        {
            _tortillaMaiz = new TortillaMaiz();
        }
        
        [TestCase]
        public void TestCurrentTemperature()
        {
            _tortillaMaiz.SetCurrentTemperature(10);
            Assert.AreEqual(10, _tortillaMaiz.GetCurrentTemperature());
        }

        [TestCase]
        public void TestFalseToast()
        {
            _tortillaMaiz.Toast(false);
            Assert.IsFalse(_tortillaMaiz.IsToasted());
        }

        [TestCase]
        public void TestTrueToast()
        {
            _tortillaMaiz.Toast(true);
            Assert.IsTrue(_tortillaMaiz.IsToasted());
        }

        [TestCase]
        public void TestToasting()
        {
            Assert.AreEqual(20, _tortillaMaiz.GetToastTemperature());
        }
    } 
}
